PennyDreadfulMTG/gatherling
# Clean-up, standardization, bug fixes, and enhancements

- Remove spurious blank line
- Apply our standard button styling on Drop Form
- Move util test into the correct directory/namespace
- We have standardized on Src suffix for these
- Standardize on rawurlencode
- Remove unused js
- Fix number of columns in empty rows in playerList
- Standardize on "link" for all hrefs and "src" for all srcs
- Form all URLs in PHP not mustache, so they can be urlencoded
- Form all URLs in PHP where they can be escaped and pass them whole to Mustache
- Escape an ampersand correctly for HTML output
- Walk back some of the strict types stuff, we're not quite there yet
- Form urls outside of mustache where they can be url escaped
- Don't form URLs for hrefs in mustache templates, it's not urlencoding safe
- Fix type error
- Add a note about how to update the test database for future tests
- Make FormatTest work at last by populating the test db with a Test Format
- Access whatsinstandard via https to avoid warning
- Be consistent in how we prevent timeouts running scripts via web
- Declare strict types some places where it seems pretty safe to do so
- Remove code that hasn't been used in a long time
- Don't echo the return value of a void function
- Cope with exactly one second time elapsed (or to go) better
- Remove file that hasn't been used in a few years
- Use approved brace style
- Use approved brace style
- Add straggling test to the namespace
- Remove parens that don't do anything
- Configure phpcs to ignore some very-frequently-violated rules for now
- Move all tests into Gatherling\Tests namespace
- Declare member var
- Mark a test that touches the db as a DatabaseCase
- Warning not error for fairly expected circumstance
- Remove obsolete commandline arg on test run
- Make updateDefaultFormats work more how it used to
- PSR-12 linting
- Start on the path to PSR-12 compliance
- PSR-12 lint
- Also teach github CI about the test datbase
- Apply fixes from StyleCI
- Add a test of Format using the new database testing stuff
- Remove pointless assignment
- "Modernize" how we do dev setup and some of the maintenance tasks
- Ignore LSP config file
- Change of approach to some of the database stuff
- Don't bother checking env if we don't care in log initialization
- Be consistent on how we import lib.php
- Some handy scripts to run with composer, docker version to follow
- Upgrade to PHPUnit 10.5
- Add a logger, using monolog, implementing the PSR LoggerInterface
- Mustachify cardsetDropMenu
- Don't dump autoload separate to running composer install
